api/src/services/project-service.ts

```diff
@@ -25,6 +25,16 @@ import { IUpdateProject } from '../paths/project/{projectId}/update';
 import { queries } from '../queries/queries';
 import { DBService } from './service';
 
+export type ProjectObject = {
```

Comment:
This is nice step, can we actually move this in `models/project-view.ts` or something as a interface and sanitary class which will make sure it is sent correctly everytime.

Reply:
yup, good call